lib/formatters/checkstyle.js

```diff
@@ -41,7 +41,8 @@ function xmlEscape(s) {
                 return "&quot;";
             case "'":
                 return "&apos;";
-            // no default
+            default:
```

Comment:
Because new `consistent-return` rule had been warning this function: `Expected to return a value at the end of this function.`